coralproject/cay
# Coral bocoup

## What does this PR do?

This pr introduces two props to the into the form creator:
- groupSubmissions on multiple choice questions indicates that submissions will be grouped by the answer to this question when doing aggregations. In doing so, it allows us to understand how people _who answered this question in each way_ answered the other questions in the form.
- includeInGroups indicates that text based responses be included in group aggregations. This allows us to control which text fields will be grouped (organized into lists).
## How do I test this PR?
- Go to the Create / Edit form interface
- Add a text area.
- Ensure that the Group Submissions checkbox sets props correctly.
- Add a Short Answer and Long Answer field. 
- Ensure that the Include in Groups checkbox sets into props.

@coralproject/frontend
